common/src/main/scala/explore/common/UserPreferencesQueries.scala

```diff
@@ -30,7 +30,7 @@ object UserPreferencesQueries {
   implicit class UserWidthsCreationOps(val self: UserWidthsCreation.type) extends AnyVal {
     import self._
 
-    def storeWidthPreference[F[_]: Effect](
+    def storeWidthPreference[F[_]: ApplicativeError[*[_], Throwable]](
```

Comment:
This is nice, a lot less power than `Effect`

Reply:
Right, and you need a `Dispatcher` to run it.